packages/frontend-vitals-monitoring/README.md

```diff
@@ -0,0 +1,149 @@
+<!-- Improved compatibility of back to top link: See: https://github.com/othneildrew/Best-README-Template/pull/73 -->
+
+<a name="readme-top"></a>
+
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  
+<h3 align="center">GOV.UK Frontend Vitals Signs</h3>
+  <p align="center">
+    This package enables GOV UK LOGIN frontend Node.js applications add frontend vitals monitoring.
```

Comment:
This sentence doesn't quite scan right to me? Should it be something like "This package enables GOV UK LOGIN frontend Node.js applications to add frontend vital signs monitoring."?

Reply:
thank you, have updated